For some reason texinfo 5 doesn't like @if... just after @macro and @end ... just before @end macro
(strange things happen...)
--- enblend-enfuse-4.1.1/doc/auxmac.texi.orig	2013-07-20 18:07:37.617634914 +0200
+++ enblend-enfuse-4.1.1/doc/auxmac.texi	2013-07-20 18:09:19.710963959 +0200
@@ -82,6 +82,7 @@
 
 @c Generate a nice representation of base^exponent.
 @macro power{base, exponent}
+
 @ifinfo
 \base\^\exponent\
 @end ifinfo
@@ -94,10 +95,12 @@
 @docbook
 \base\<superscript>\exponent\</superscript>
 @end docbook
+
 @end macro
 
 
 @macro classictimes
+
 @ifinfo
 x@c gobble following newline  -- The Tricks of a Texinfo Wizard.
 @end ifinfo
@@ -110,6 +113,7 @@
 @docbook
 &times;
 @end docbook
+
 @end macro
 
 
@@ -117,6 +121,7 @@
 @c for TeX lives in auxmac.tex.
 @ifnottex
 @macro geq
+
 @ifinfo
 >=@c
 @end ifinfo
@@ -126,6 +131,7 @@
 @docbook
 &ge;
 @end docbook
+
 @end macro
 @end ifnottex
 
@@ -140,6 +140,7 @@
 @c for TeX lives in auxmac.tex.
 @ifnottex
 @macro leq
+
 @ifinfo
 <=@c
 @end ifinfo
@@ -149,6 +150,7 @@
 @docbook
 &le;
 @end docbook
+
 @end macro
 @end ifnottex
 
@@ -172,6 +174,7 @@
 @c Special Characters
 
 @macro inlineomega
+
 @ifinfo
 @math{omega}@c
 @end ifinfo
@@ -186,10 +189,12 @@
 @docbook
 &omega;
 @end docbook
+
 @end macro
 
 
 @macro inlinesigma
+
 @ifinfo
 @math{sigma}@c
 @end ifinfo
@@ -204,10 +209,12 @@
 @docbook
 &sigma;
 @end docbook
+
 @end macro
 
 
 @macro inlinexi
+
 @ifinfo
 @math{xi}@c
 @end ifinfo
@@ -222,6 +229,7 @@
 @docbook
 &xi;
 @end docbook
+
 @end macro
 
 
@@ -239,6 +245,7 @@
 
 
 @macro semilog{significant, exponent}
+
 @ifinfo
 \significant\*10^\exponent\@c
 @end ifinfo
@@ -258,4 +265,5 @@
 @docbook
 \significant\&times;10<superscript>\exponent\</superscript>
 @end docbook
+
 @end macro
--- enblend-enfuse-4.1.1/doc/enfuse.texi.orig	2013-07-20 18:14:21.924284611 +0200
+++ enblend-enfuse-4.1.1/doc/enfuse.texi	2013-07-20 18:33:00.144237684 +0200
@@ -2434,6 +2434,7 @@
 input images@tie{}@math{n}.
 
 @macro equationW{}
+
 @ifnotdocbook
 @ifnottex
 (W)
@@ -2445,6 +2446,7 @@
 @docbook
 <xref linkend="equ:pixel-weighting-function"/>
 @end docbook
+
 @end macro
 
 Enfuse allows for weighting the contribution of each @math{P(i, x, y)}
--- enblend-enfuse-4.0-753b534c819d/doc/enblend.texi.orig	2009-12-21 08:22:30.000000000 +0100
+++ enblend-enfuse-4.0-753b534c819d/doc/enblend.texi	2010-10-31 13:27:08.515099937 +0100
@@ -28,10 +28,10 @@
 @c Categorization for the GNU Info System
 @c
 
-@dircategory Individual utilities
+@dircategory Graphics Utilities:
 
 @direntry
-* enblend: (enblend).                   Blend images using a multiresolution spline.
+* enblend: (enblend).			Blend images using a multiresolution spline
 @end direntry
 
 
--- enblend-enfuse-4.0-753b534c819d/doc/enfuse.texi.orig	2009-12-21 08:22:30.000000000 +0100
+++ enblend-enfuse-4.0-753b534c819d/doc/enfuse.texi	2010-10-31 13:27:25.587104966 +0100
@@ -28,10 +28,10 @@
 @c Categorization for the GNU Info System
 @c
 
-@dircategory Individual utilities
+@dircategory Graphics Utilities:
 
 @direntry
-* enfuse: (enfuse).                     Fuse images using a multiresolution spline.
+* enfuse: (enfuse).			Fuse images using a multiresolution spline
 @end direntry
 
 
